framework/tests/signature-inference/non-annotated/InheritanceTest.java

```diff
@@ -0,0 +1,28 @@
+//@skip-test
+// TODO: Enable this test when signature inference is working correctly for cases
```

Comment:
Do you plan to file an issue for this? Or will this be fixed before we merge this PR?